config/locales/in_person_proofing/es.yml

```diff
@@ -143,13 +143,11 @@ es:
         state_id_jurisdiction_hint: Este es el estado que emitió su identificación
         state_id_jurisdiction_prompt: '- Seleccione -'
         state_id_number: Número de identidad
-        state_id_number_hint_html: "Puede incluir letras, números y los siguientes
-          símbolos: <span class='usa-sr-only'>espacios, </span><span
-          aria-hidden='true'> </span> <span class='usa-sr-only'>barras
-          diagonales, </span> <span aria-hidden='true'>/</span> <span
-          class='usa-sr-only'>asteriscos, </span><span
-          aria-hidden='true'>*</span> <span
-          class='usa-sr-only'>guiones</span><span aria-hidden='true'>-</span>"
+        state_id_number_hint: 'Puede incluir letras, números y los siguientes símbolos:'
+        state_id_number_hint_asterisks: 'asteriscos'
```

Comment:
Do the apostrophes matter here? I just noticed they are missing from the English translation.

Reply:
do you mean the quotes around the word `'asteriscos'`? I think they come from `make normalize_yaml`

Reply:
I do yes. Ah ok. Well I'm ok with that.